src/main/java/org/example/datastructures/arrays/MergeTwoSortedArray.java

```diff
@@ -0,0 +1,48 @@
+Implement the method int[] mergeArrays(int[] arr1, int[] arr2) that takes two chronologically sorted arrays and returns a new sorted array including all elements from the input arrays.
+
+class checkMergeArray { 
```

Comment:
- Use CamelCase(first letter of each word capital) for class name
- Add package info at top of class as
 ```java
package org.example.datastructure.arrays;
```